src/cdk/stepper/stepper.ts

```diff
@@ -30,6 +30,12 @@ import {AbstractControl} from '@angular/forms';
 /** Used to generate unique ID for each stepper component. */
 let nextId = 0;
 
+/**
+ * Position state of the content of each step in horizontal stepper that is used for transitioning
+ * the content into correct position upon step selection change.
+ */
+export type CdkStepContentPositionState = 'left' | 'center' | 'right';
```

Comment:
Can this be generalized such that it isn't only for horizontal steps? Maybe `previous` /  `current` / `next`?